### DISCUSSION TITLE
How to change Theme colors default

### DISCUSSION THREAD

**Discussion by Enat0** [HandyOrg/HandyControl#1133] [Q&A] [answered]
Hello.

I'm trying to change some default colors but can't seem to do it..

How can I change the default dark/light color for the header for example? I can't seem to find the documentation or the correct key..

Also how can I change the selected item color in datagrid/listview  after lost focus? I want to maintain the same color but it changes to gray:

Example (Left is already selected and I'm selecting the right one):
![image](https://user-images.githubusercontent.com/100444259/157496431-bc34c738-655d-4ddc-9258-4e2e002b4a9f.png)

My App.xaml:

```
<ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <hc:ThemeResources>
                    <hc:ThemeResources.ThemeDictionaries>
                        
                        <ResourceDictionary x:Key="Light" hc:ThemeDictionary.Key="Light">
                            <ResourceDictionary.MergedDictionaries>
                                <ResourceDictionary Source="Themes/Light.xaml"/>
                            </ResourceDictionary.MergedDictionaries>
                        </ResourceDictionary>

                        <ResourceDictionary x:Key="Dark" hc:ThemeDictionary.Key="Dark">
                            <ResourceDictionary.MergedDictionaries>
                                <ResourceDictionary Source="Themes/Dark.xaml"/>
                            </ResourceDictionary.MergedDictionaries>
                        </ResourceDictionary>
                        
                    </hc:ThemeResources.ThemeDictionaries>
                </hc:ThemeResources>
                <hc:Theme/>
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
```

My Light Theme:

```
<ResourceDictionary
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:hc="https://handyorg.github.io/handycontrol">
    
    <ResourceDictionary.MergedDictionaries>
        <hc:ColorPaletteResources
            TargetTheme="Light"
            PrimaryBrush="#2db84d"
            DangerColor="#7b1fa2"
            BackgroundColor="Pink"
            MainContentBackgroundColor="Pink"
            />
    </ResourceDictionary.MergedDictionaries>

    <!-- IMAGENS-->
    <ImageBrush x:Key="logoABFundo" ImageSource="../Imagens/Light/Logo_AB.png" Opacity="0.3" Stretch="None"/>
    <BitmapImage x:Key="logoABAssistencias" UriSource="../Imagens/Componentes/Logo.png"/>
    
</ResourceDictionary>
```

**Comment by ghost1372** [comment, marked as answer]
use this doc
https://ghost1372.github.io/handycontrol/theme/colorPreset/
or  this
https://ghost1372.github.io/handycontrol/theme/#ThemeDictionaries-Override-predefined-colors

you can find colors/brushes key in source code

https://github.com/HandyOrg/HandyControl/tree/master/src/Shared/HandyControl_Shared/Themes/Styles

**Reply by Enat0** [reply]
Thanks, I was following the docs you mentioned, but didn't know I had to use the "RegionBrush" property.

I'd suggested adding the link "https://github.com/HandyOrg/HandyControl/tree/master/src/Shared/HandyControl_Shared/Themes/Styles" to the colorpreset or themedictionary page for easy access
